### DISCUSSION TITLE
How to turn off comments from merge_protection rules on the PR page?

### DISCUSSION THREAD

**Discussion by bartgol** [Mergifyio/mergify#5129] [Q&A] [answered]
The merge protection rule is adding a comment on the PR page, which is pointless, since the info on the checks is readily available in the checks tab.

I tried to add "comment: false" to the `merge_protections` section as well as to the rule section, but it errored out saying it's not in the schema.

Am I not using the correct syntax? Or is this not possible? If not possible, would it make sense to add the feature?

**Comment by jd** [comment]
👋 @bartgol 
This is not currently possible, but I'll add this as a feature request.

**Reply by marpulli** [reply]
I'd also love this feature if it gets implemented!

**Reply by jd** [reply, marked as answer]
I just vibe coded the feature: https://changelog.mergify.com/changelog/allow-to-disable-comment-posting

**Comment by jd** [comment]
I just
